src/saving/SaveHelper.cs

```diff
@@ -116,18 +116,36 @@ public static void LoadSave(string name)
     /// <summary>
     ///   Loads the save file with the latest write time
     /// </summary>
-    public static void QuickLoad()
+    /// <returns>False if the versions do not match</returns>
```

Comment:
I think this should mention that if this returns false, then the save won't be loaded.